src/cldfbench/commands/media.py

```diff
@@ -34,6 +34,7 @@
 import zipfile
 import threading
 import collections
+import rfc3986
```

Comment:
This import should go into the second group of imports, see https://www.python.org/dev/peps/pep-0008/#imports